packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -379,7 +379,6 @@ function ShellHeader() {
             <div className="mb-1 h-6 w-32 animate-pulse rounded-md bg-gray-200" />
           )}
         </div>
-        <div className="ml-auto">{meta.CTA}</div>
```

Comment:
Why was this removed?

Reply:
i can't say why i think it was just a mistake. i have added it back